_includes/markdown/Markup.md

```diff
@@ -285,10 +285,67 @@ Combining automated and manual testing practices allows 10up to maintain a high
 
 ### Feature Detection
 At 10up, the concept of feature detection is used to test browser support for new features that do not yet have full support across the board. The concept of feature detection is to test if a feature is supported by the browser and if not supported, conditionality run code to provide a similar experience with browsers that do support the feature. While popular [feature detection libraries](https://modernizr.com/) exist, there are [feature detection techniques](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#JavaScript) for JavaScript and [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) at-rule for CSS that can be utilized.
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. Here are a few known benefits of SVG:
+
+* __Scalability__ - They look great on retina displays and at any size, i.e. they're resolution independent.
+* __File Size__ - Small file size and compresses well.
+* __Styling__ - Manipulate fill, stroke, and even animate.
+
+### SVG Sprites
+
+It is optimal to group multiple SVG icons that are likely to be used throughout certain templates or even the entire site (i.e. sprite).
+
+Combine them in to an `svg-defs.svg` file, and reference them within the template source with `<use>`. The creation of this icon system should be automated.
+
+Be sure to hide the `svg-defs.svg` from accessibility technologies with `display: none;`.
+
+
+### SVG embedded in HTML
+
+When your placing an SVG in markup (i.e. inline) be sure to use the following guidelines:
+
+* If the SVG is purely **decorative**: 
+    * An empty `alt=""` can be used: `<img alt="">`, or
+	* Use ARIA to hide the element from assistive technologies: `<svg aria-hidden="true">`
+* If the SVG is **meaningful** then use `<title>` and possibly even `<desc>` to describe the graphic.  Also, be sure to add an `id` to each element, and appropriate ARIA to overcome a known bug in [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=231654&q=SVG%20%20title%20attribute&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified) and [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1151648).
+
+	```html
+	<!-- role="img" to exclude image from being traversed by certain browsers w/ group role -->
+	<svg role="img" aria-labelledby="uniqueTitleID uniqueDescID">
+		<title id="uniqueTitleID">The Title</title>
+		<desc id="uniqueDescID">Description goes here...</desc>
+	</svg>
+	```
+
+* Use `aria-label` if the SVG is linked and has no supporting text.
+
+	```html
+	<a href="#" aria-label="See Twitter feed">
```

Comment:
@colorful-tones using # isn't accessible, I know this is just an example, but let's put the 10up twitter in here

Reply:
I replaced the hash link with 10up's Twitter URL. Thanks!